pep-dortmund/corporate-design
# Added python script to create a wordcloud

__How to test__
First, make sure that you ran the Makefile of the `logos` directory.
Afterwards, go into the `PeP_wordcloud` directory and run `python pep_wordcloud.py`
